Energinet-DataHub/greenforce-frontend
# chore: update NX to latest version

## Description


### Workspace
- update NX to latest version
- update Angular and related packages to v14
- update Playwright to latest version
- update NOTICE file
- run migrations to adjust files
  - add `$schema` to files
  - set target to `es2020` for libraries
  - use untyped form controls
  - adjust configuration in Nx file

### DataHub
- adjust path for Watt assets
- fix custom Playwright executor

### Energy Origin
- broaden error message check in e2e tests
- adjust path for Watt assets
- bump project version

### Watt
- delete temporary `ui-watt` assets directory
- fix errors related to typed form controls
- fix Storybook types
- remove unnecessary `entryComponents` property

## References



- #563
